packages/@aws-cdk/aws-secretsmanager/test/test.secret.ts

```diff
@@ -68,6 +68,109 @@ export = {
     test.done();
   },
 
+  'secret with kms'(test: Test) {
```

Comment:
I think the changes in the existing tests cover this well enough... I don't think we need an additional test just for this.

Reply:
That's fair. I'll blow that away.